src/Server/Formatter.php

```diff
@@ -0,0 +1,63 @@
+<?php
+namespace LanguageServer\Server;
+
+use LanguageServer\Protocol\TextEdit;
+use LanguageServer\Protocol\Range;
+use LanguageServer\Protocol\Position;
+
+class Formatter
+{
+
+    /**
+     * @param string $content
+     * @param string $uri
+     *
+     * @return TextEdit[]
+     */
+    public function format(string $content, string $uri)
+    {
+        // remove 'file://' prefix
+        $uri = substr($uri, 7);
```

Comment:
For now I added new method to Formatter to avoid conflicts with #31. Later I can move it to utils.php